build.ps1

```diff
@@ -60,6 +60,18 @@ function NugetPack {
     }
 }
 
+function PackSelfContainProject {
```

Comment:
...

---
In reply to: [95931939](https://github.com/dotnet/docfx/pull/1167#discussion_r95931939) [](ancestors = 95931939,95931418)